src/XrdHdfs.cc

```diff
@@ -695,6 +695,7 @@ int XrdHdfsFile::Read(XrdSfsAio *aiop)
 {
 
 // Execute this request in a synchronous fashion
+// Will not be called as XrdHdfsSys::Features() advertises no AIO support.
```

Comment:
You may want to mention pgread() since this is the default behavior for that one as well as there is a default implementation in the base class XrdOss.